website/pages/docs/concepts/seal.mdx

```diff
@@ -89,75 +89,159 @@ Recovery Keys and are used for these operations.
 For a list of examples and supported providers, please see the
 [seal documentation](/docs/configuration/seal).
 
+## Recovery Key Rekeying
+
+During KMS Seal initialization process, a set of Shamir keys called Recovery Keys are
+generated which are used for operations that still require a quorum of users.
+
+Recovery Keys can be rekeyed to change the number of shares or thresholds. When using the
+Vault CLI, this is performed by using the `-target=recovery` flag to `vault operator rekey`.
+
 ## Seal Migration
 
-The seal can be migrated from Shamir keys to Auto Unseal, Auto Unseal to Shamir
-keys and Auto Unseal to another Auto Unseal.
+The seal can be migrated from Shamir Seal to KMS Seal, KMS Seal to Shamir Seal
+and KMS Seal to another KMS Seal.
+
+~> **NOTE**: The seal migration process cannot be performed without downtime.
+Due to the intricate technical underpinnings of the seal feature, it is at this
```

Comment:
I would delete "at this point" but couldn't figure out how to do a multiline suggestion.

Reply:
Done.

When clicking the `+` button, you hold and drag it down first to select multiple lines and then click it to do a multi line comment.